### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by afcastaneda223**
# Documentation

All the documentation is on the README file.

## Updates:

- test for the function for editing the task description.
-  test for the function for updating an item's 'completed' status.
-  test for the function for updating an item's index value upon drag/drop actions.
-  test for the "Clear all completed" function.

## Notes:

Coded by @afcastaneda223 @alonsofl 

✅  - No linters errors were detected. 
✅  - Used Javascript for functions. 
✅  - Todo list was created dynamically.

### CODE REVIEW

File: src/edit-status-index-clear.test.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      description: 'read',
+      index: 0,
+      completed: false,
+    },
+  ];
+
+  test('swap first for third index', () => {
+    expect(removeChecked()).toEqual(expectedOne);
+  });
+});
```
Comment (DeliceLydia) [inline src/edit-status-index-clear.test.js]: - [ ] Please add some tests on `Clear all completed function` try to test negative and positive scenarios

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/edit-status-index-clear.test.js (hunk 1 above)
Comment (afcastaneda223) [reply src/edit-status-index-clear.test.js]: done

**Review by carlos-ssh — APPROVED** [review]
Hi @afcastaneda223, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
